src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,14 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
 public class Country implements Cloneable {
```

Comment:
Add `@Table`.
By default the table name is the entity name, but it would be great to make it more clear.